packages/devextreme/js/__internal/ui/chat/messagelist.ts

```diff
@@ -325,6 +338,13 @@ class MessageList extends Widget<Properties> {
         super._optionChanged(args);
     }
   }
+
+  getEmptyViewId(): string | null {
+    const $emptyView = this.$element().find(`.${CHAT_MESSAGELIST_EMPTY_VIEW_CLASS}`);
```

Comment:
Why didn't we check the data here and use a search by the DOM tree? Something like this

```
return this._isEmpty() ? $emptyView.attr('id') : null;
```
_isEmpty() is reliable indicator that the empty view element is currently present in the DOM tree. 
 Another reason to use this approach is performance. If a lot of messages are rendered in the chat, we would need to spend time processing each element.
If you encounter any difficulties with this approach, feel free to let me know

Reply:
Okay, I will refactor it, but don't we then want to get rid of .find() in all places in our code to avoid Depth-First Search?

Reply:
[604f5dc44f8c0d730eca6fcecfa778976849c4d0]